lib/code_corps/github/adapters/task.ex

```diff
@@ -39,5 +43,33 @@ defmodule CodeCorps.GitHub.Adapters.Task do
     |> Map.from_struct
     |> MapTransformer.transform_inverse(@mapping)
     |> Map.drop(@autogenerated_github_keys)
+    |> add_user_info_header(task)
   end
+
+  @separator "\r\n\r\n---\r\n\r\n"
```

Comment:
This could probably be configurable.

There is also the problem that it isn't necessarily unique. Issues and comments could have this separator anyway, so we should have additional logic to ensure the first section (before the first separator) is actually our header section.

We can't compute the entire section and check for it, since we just have  payload, not the actual comment/task record.

Reply:
We should definitely keep that visible separator but we could also hard end with a Markdown comment:

```
\r\n\r\n[//]: # (Please type your edits below this line)\r\n---\r\n\r\n
```